src/proxy-ui-api/frontend/src/views/Service/Endpoints/Endpoints.vue

```diff
@@ -77,8 +82,16 @@ export default Vue.extend({
   },
   computed: {
     ...mapGetters(['service']),
+
+    endpoints(): Endpoint[] {
+      const filtered = this.service.endpoints.filter((endpoint: Endpoint) => {
+        return !this.isBaseEndpoint(endpoint);
+      });
+
+      return filtered;
```

Comment:
one more thing. there's no need to create that filtered -variable. Just return the value from expression.

Reply:
fixed